src/OpenTelemetry.AutoInstrumentation/Loading/Initializers/QuartzInitializer.cs

```diff
@@ -0,0 +1,43 @@
+// <copyright file="QuartzInitializer.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using OpenTelemetry.AutoInstrumentation.Plugins;
+
+namespace OpenTelemetry.AutoInstrumentation.Loading.Initializers;
+
+internal class QuartzInitializer : InstrumentationInitializer
+{
+    private readonly PluginManager _pluginManager;
+
+    public QuartzInitializer(PluginManager pluginManager)
+        : base("Quartz")
+    {
+        _pluginManager = pluginManager;
+    }
+
+    public override void Initialize(ILifespanManager lifespanManager)
+    {
+        var instrumentationType = Type.GetType("OpenTelemetry.Instrumentation.Quartz.QuartzJobInstrumentation, OpenTelemetry.Instrumentation.Quartz")!;
+
+        var options = new OpenTelemetry.Instrumentation.Quartz.QuartzInstrumentationOptions();
+
+        _pluginManager.ConfigureTracesOptions(options);
+
+        var instrumentation = Activator.CreateInstance(instrumentationType, options)!;
+
+        lifespanManager.Track(instrumentation);
```

Comment:
nit: why these empty lines?

Reply:
_Inspired_ by https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/ee07b6ea2d0c61949531870b2bff9a24ae7f8780/src/OpenTelemetry.AutoInstrumentation/Loading/Initializers/GrpcClientInitializer.cs#L34-L45

I would keep as it is.